src/frontend/komponenter/Skjema/SelectMedOptions.tsx

```diff
@@ -1,19 +1,20 @@
 import React, { FC } from 'react';
 
-import Select, { SelectProps } from './Select';
+import { Select, SelectProps } from '@navikt/ds-react';
 
 export interface SelectOption {
     value: string;
     label: string;
 }
 
-interface Props extends Omit<SelectProps, 'children'> {
+interface Props extends Omit<SelectProps, 'children' | 'readOnly'> {
     valg: SelectOption[];
+    erLesevisning: boolean;
 }
 
-const SelectMedOptions: FC<Props> = ({ valg, ...props }) => {
+const SelectMedOptions: FC<Props> = ({ valg, erLesevisning, ...props }) => {
     return (
-        <Select {...props}>
+        <Select {...props} readOnly={erLesevisning}>
```

Comment:
Burde vi bruke noe annet enn readOnly ved lesevisning? Som i `DateInputMedLeservisning`

Reply:
Dette er gjort bevisst på akkurat denne sånn at feltene blir like. Så veldig dumt ut med label og leservisning ved på kun _ytelse/situasjon_ når feltene ved siden av var høyere. Denne versjonen brukes også kun i redigeringsmodus så beslutter eller andre som er inne vil ikke se denne versjonen:

<img width="521" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/c95b0140-9fa4-4bd3-9c76-73a4e05a38b1">

Ser nå at den samme komponenten brukes i stønadsperioder så der ser det litt dummere ut så må fikse de (selv om designet her skal endres): 
<img width="1289" alt="image" src="https://github.com/navikt/tilleggsstonader-sak-frontend/assets/46678893/5a8287d0-4c73-4bcf-b99a-94031c255a6b">

Reply:
Hva med noe som [dette](https://github.com/navikt/tilleggsstonader-sak-frontend/commit/9d32f16c39235911c18e1e473ee08bd372533f3e)? Så sender man inn som prop om leservisningen skal være som `readOnly`

Reply:
Ah. Då skjønner jeg hvorfor du brukt readOnly
Tenker det er forskjell på bruket av "erLesevisning" og "readOnly"
Iblant har man behov for det ene, og iblant det andre? Så burde kanskje ikke blande de?